fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/HiveCompatibleCatalog.java

```diff
@@ -57,7 +57,7 @@ public void initialize(String name, FileIO fileIO,
     protected FileIO initializeFileIO(Map<String, String> properties, Configuration hadoopConf) {
         String fileIOImpl = properties.get(CatalogProperties.FILE_IO_IMPL);
         if (fileIOImpl == null) {
-            FileIO io = new S3FileIO();
+            FileIO io = new HadoopFileIO(hadoopConf);
```

Comment:
s3 file need some custom configuration, so hadoop io is used in superclass by default, we can add better implementations to derived class just like the implementation in dlf catalog
.